src/app/aircraft/add/index.njk

```diff
@@ -6,8 +6,10 @@
 	{{__('error_prefix') if errors}} GAR | Save an Aircraft
 {% endblock %}
 
+{% set active = '/aircraft' %}
```

Comment:
This nunjucks "active" variable is used by the header links as the logic to determine whether it should be highlighted (set to active), by comparing it to the href it points to.

Simplistic, but reasonable efficient approach.